operator-common/src/main/java/io/strimzi/operator/common/Util.java

```diff
@@ -369,4 +371,24 @@ public static ConfigResource getBrokersConfig(int podId) {
     public static ConfigResource getBrokersConfig(String podId) {
         return new ConfigResource(ConfigResource.Type.BROKER, podId);
     }
+
+    public static String getStringHash(String toBeHashed)  {
+        try {
+            MessageDigest hashFunc = MessageDigest.getInstance("SHA-512");
```

Comment:
512 bit hash => 64 bytes => 128 chars in the string. I don't think we need to be so resilient to a collision that we need to spend that much effort. 64 bits would be more than enough IMHO.